src/main/java/net/mcreator/workspace/WorkspaceFolderManager.java

```diff
@@ -57,6 +57,10 @@ public ImageIcon getItemImageIcon(String textureIdentifier) {
 		return new ImageIcon(getItemTextureFile(textureIdentifier).getAbsolutePath());
 	}
 
+	public ImageIcon getPotionEffectImageIcon(String textureIdentifier) {

```

Comment:
Keep this system as it is for now, so texture from other textures and re-mapped by generator into potion_effects, as how it was for now and paintings do this. Later PR with Goldorion's texture categories will change this

Reply:
I will start working on converter once this is fixed as at this point I can take over PR to add converter, and then you can continue your work @Max094Reikeb